pkg/scheduler/framework/plugins/serviceaffinity/service_affinity.go

```diff
@@ -18,22 +18,28 @@ package serviceaffinity
 
 import (
 	"context"
+	"fmt"
 
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"
+	"k8s.io/kubernetes/pkg/scheduler/algorithm/priorities"
 	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/migration"
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 	"k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 )
 
-// Name of this plugin.
+// Name is the name of the plugin used in the plugin registry and configurations.
 const Name = "ServiceAffinity"
 
 // Args holds the args that are used to configure the plugin.
 type Args struct {
-	// Labels should be present for the node to be considered a fit for hosting the pod
+	// Labels are homogeneous for pods that are scheduled to a node.
+	// (i.e. it returns true IFF this pod can be added to this node such that all other pods in
+	// the same service are running on nodes with the exact same values for Labels).
 	Labels []string `json:"labels,omitempty"`
+	// AntiAffinityLabelsPreference are the labels to consider for service anti affinity scoring.
+	AntiAffinityLabelsPreference []string `json:"antiAffinityLabelsPreference,omitempty"`
```

Comment:
should we name the variables more consistently? like AffinityLabels and AntiAffinityLables?

Reply:
renamed as you suggeste